docs/src/main/asciidoc/mongodb-panache-guide.adoc

```diff
@@ -381,6 +381,30 @@ public class Person extends PanacheMongoEntity {
 Both `findByNameWithPanacheQLQuery()` and `findByNameWithNativeQuery()` methods will return the same result but query written in PanacheQL
 will use the entity field name: `name`, and native query will use the MongoDB field name: `lastname`.
 
+== Query projection
+
+Query projection can be done with the `project(Class)` method on the `PanacheQuery` object that is returned by the `find()` method.
+You can use it to restrict which fields will be returned by the database, for this you need to create a class (a Pojo) that will only contains the projected fields.
```

Comment:
The implementation checks that there is a getter for that field, so for clarity I would add "with the corresponding getter" or something. 
Also, contains -> contain

Reply:
As public entity fields are used a lot accross Quarkus examples, maybe I also need to implements the retieval of these fields. The Pojo Codec that is used supports them so it should be done.

Reply:
Public fields and getters/setters are now supported.